doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.md

```diff
@@ -28,6 +28,7 @@ To understand the following content better, you should understand the difference
 
 * RayCluster configuration
   * `rayClusterSpec` - Defines the **RayCluster** custom resource to run the Ray job on.
+  * `clusterSelector` - Use existing **RayCluster** custom resources to run the Ray job instead of creating a new one. See [ray-job.use-existing-raycluster.yaml](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-job.use-existing-raycluster.yaml) for example configurations.
```

Comment:
I think this doesn't belong to `rayClusterSpec`. Maybe move to "Submission configuration" instead?

Reply:
Either `rayClusterSpec` or `clusterSelector` should be set right? So I think it is better to put here, because it configures the underlying RayCluster.

Reply:
https://github.com/ray-project/kuberay/blob/55a66880a6128415206468fe785a3cd79c50874e/ray-operator/controllers/ray/rayjob_controller.go#L827-L829

Reply:
I knew that. I expect that user's initial impression was that fields under "RayCluster configuration" should relate to the RayCluster CRD. For example, `rayClusterSpec` corresponds to `RayClusterSpec`. However, `clusterSelector` is not a field in RayCluster CRD. In addition, it is used to determine where the job should submit to.

That's why I think moving it to 'Submission configuration' is better.

Your point makes sense to me as well, and since it also includes a YAML example, I’m fine with leaving it in the current place.